### PR TITLE
Add "Last changed by" to repo page

### PR DESCRIPTION
**Pull request by wozjac**
Proposition for #3267

### CODE REVIEW

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap [hunk 1]
```diff
@@ -953,6 +960,12 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_VIEW IMPLEMENTATION.
     ri_html->add( render_item_files( is_item ) ).
     ri_html->add( '</td>' ).
 
+    IF mo_repo->has_remote_source( ) = abap_true.
```
Comment (mbtools) [inline src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:963]: why only for remote? it should also work for "zip repos"

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap [hunk 2]
```diff
@@ -1095,6 +1108,13 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_VIEW IMPLEMENTATION.
     ls_col_spec-allow_order_by = abap_true.
     APPEND ls_col_spec TO lt_col_spec.
 
+    IF mo_repo->has_remote_source( ) = abap_true.
```
Comment (mbtools) [inline src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:1111]: same

**Comment by mbtools** [comment]
I like it. Performance for repos with a lot of changed objects might become an issue (`changed_by` typically is a new DB access) but I think it's uncommon to have hundreds of changes.

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap (hunk 2 above)
Comment (wozjac) [reply src/ui/zcl_abapgit_gui_page_repo_view.clas.abap]: Hi, 
commited new version for local ones too.

**Review by mbtools — APPROVED** [review]
👍

**Comment by larshp** [comment]
@wozjac thanks, I've added you as a collaborator, you should now have access to create branches in this repository 🎉

**Comment by larshp** [comment]
screenshot,
image

**Comment by wozjac** [comment]
Thank you!

**Pull request merged**